quartz/plugins/transformers/ofm.ts

```diff
@@ -17,6 +17,9 @@ import { toHtml } from "hast-util-to-html"
 import { PhrasingContent } from "mdast-util-find-and-replace/lib"
 import { capitalize } from "../../util/lang"
 import { PluggableList } from "unified"
+import yaml from "js-yaml"
+import toml from "toml"
+import matter from "gray-matter"
```

Comment:
You actually don't have to do any of these parsing.

Probably will only need to parse the links within `markdownPlugins`.

you can get the frontmatter from `file.frontmatter`.

Reply:
This means the nested frontmatter links would also getting parsed afaik.

probably should set the options by default.